### ISSUE TITLE
Is there a way to combine Control and Control-Camera models?

### ISSUE THREAD

**Issue by codeysun** [aigc-apps/VideoX-Fun#351]
Hi, thank you for the great work. Is there a way to combine the Control and Control-Camera models to have both video control and camera control? Could I, e.g., copy the weights of the camera encoder onto the Control model? Thanks!

**Comment by bubbliiiing** [comment]
It doesn't seem feasible right now. May I ask if there's an actual need for this? What's the specific use case?

**Comment by Xushuolin** [comment]
> It doesn't seem feasible right now. May I ask if there's an actual need for this? What's the specific use case?

我训练之后发现不太可行，输入一段pose 控制角色跳舞，另外输入一段相机轨迹控制镜头移动，会出现场景是正常符合相机移动轨迹的，但是人物留在原地不动仅符合pose的控制。（比如一段平移镜头）

**Comment by codeysun** [comment]
> It doesn't seem feasible right now. May I ask if there's an actual need for this? What's the specific use case?

I have a pose control that comes from a video with a moving camera. When I generate with only pose control, the background does not properly move (it stays still even though it should be moving). I think that adding camera controls will solve this issue.

**Comment by codeysun** [comment]
> > It doesn't seem feasible right now. May I ask if there's an actual need for this? What's the specific use case?
> 
> 我训练之后发现不太可行，输入一段pose 控制角色跳舞，另外输入一段相机轨迹控制镜头移动，会出现场景是正常符合相机移动轨迹的，但是人物留在原地不动仅符合pose的控制。（比如一段平移镜头）

Could you share details of training and how you combined the model weights?

**Comment by bubbliiiing** [comment]
Next time I train this model, I'll consider combining the two models for joint training to see the results.


> > It doesn't seem feasible right now. May I ask if there's an actual need for this? What's the specific use case?
> 
> I have a pose control that comes from a video with a moving camera. When I generate with only pose control, the background does not properly move (it stays still even though it should be moving). I think that adding camera controls will solve this issue.

**Comment by Xushuolin** [comment]
> > > It doesn't seem feasible right now. May I ask if there's an actual need for this? What's the specific use case?
> > 
> > 
> > 我训练之后发现不太可行，输入一段pose 控制角色跳舞，另外输入一段相机轨迹控制镜头移动，会出现场景是正常符合相机移动轨迹的，但是人物留在原地不动仅符合pose的控制。（比如一段平移镜头）
> 
> Could you share details of training and how you combined the model weights?

既然你的pose和相机轨迹是从同一个视频中获取的那么这是可行的，关于训练其实很简单，由于我只有pose video的成对数据没有相机轨迹数据，因此我直接在camera_control权重上使用这些数据集进行微调，同时启用reference image。 维持control latent由pose video获得 ，因为现在camera_control和control pixel 写成互斥的了。然后我发现可行 ，在1.3b 模型上大概10000步就有效果了。

**Comment by bubbliiiing** [comment]
> > > > It doesn't seem feasible right now. May I ask if there's an actual need for this? What's the specific use case?
> > > 
> > > 
> > > 我训练之后发现不太可行，输入一段pose 控制角色跳舞，另外输入一段相机轨迹控制镜头移动，会出现场景是正常符合相机移动轨迹的，但是人物留在原地不动仅符合pose的控制。（比如一段平移镜头）
> > 
> > 
> > Could you share details of training and how you combined the model weights?
> 
> 既然你的pose和相机轨迹是从同一个视频中获取的那么这是可行的，关于训练其实很简单，由于我只有pose video的成对数据没有相机轨迹数据，因此我直接在camera_control权重上使用这些数据集进行微调，同时启用reference image。 维持control latent由pose video获得 ，因为现在camera_control和control pixel 写成互斥的了。然后我发现可行 ，在1.3b 模型上大概10000步就有效果了。

相机模型好像这做会有点退化0 0.

**Comment by Xushuolin** [comment]
> > > > > It doesn't seem feasible right now. May I ask if there's an actual need for this? What's the specific use case?
> > > > 
> > > > 
> > > > 我训练之后发现不太可行，输入一段pose 控制角色跳舞，另外输入一段相机轨迹控制镜头移动，会出现场景是正常符合相机移动轨迹的，但是人物留在原地不动仅符合pose的控制。（比如一段平移镜头）
> > > 
> > > 
> > > Could you share details of training and how you combined the model weights?
> > 
> > 
> > 既然你的pose和相机轨迹是从同一个视频中获取的那么这是可行的，关于训练其实很简单，由于我只有pose video的成对数据没有相机轨迹数据，因此我直接在camera_control权重上使用这些数据集进行微调，同时启用reference image。 维持control latent由pose video获得 ，因为现在camera_control和control pixel 写成互斥的了。然后我发现可行 ，在1.3b 模型上大概10000步就有效果了。
> 
> 相机模型好像这做会有点退化0 0.

会有一点退化 哈哈，可以用Humanvid 数据集去训练 他们提供了，一些和pose视频成对的相机轨迹

**Comment by codeysun** [comment]
> > > > It doesn't seem feasible right now. May I ask if there's an actual need for this? What's the specific use case?
> > > 
> > > 
> > > 我训练之后发现不太可行，输入一段pose 控制角色跳舞，另外输入一段相机轨迹控制镜头移动，会出现场景是正常符合相机移动轨迹的，但是人物留在原地不动仅符合pose的控制。（比如一段平移镜头）
> > 
> > 
> > Could you share details of training and how you combined the model weights?
> 
> 既然你的pose和相机轨迹是从同一个视频中获取的那么这是可行的，关于训练其实很简单，由于我只有pose video的成对数据没有相机轨迹数据，因此我直接在camera_control权重上使用这些数据集进行微调，同时启用reference image。 维持control latent由pose video获得 ，因为现在camera_control和control pixel 写成互斥的了。然后我发现可行 ，在1.3b 模型上大概10000步就有效果了。

Thank you! So you are fully fine-tuning the camera encoder, and freezing everything else? Do you have information on dataset size, learning rate, and batch size?
